src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/validator/CreateItemRequestValidator.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.validator;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.certifiedcopies.orders.api.dto.CertifiedCopyItemOptionsRequestDTO;
+import uk.gov.companieshouse.certifiedcopies.orders.api.dto.CertifiedCopyItemRequestDTO;
+import uk.gov.companieshouse.certifiedcopies.orders.api.model.CertifiedCopyItem;
+import uk.gov.companieshouse.certifiedcopies.orders.api.model.CertifiedCopyItemOptions;
+import uk.gov.companieshouse.certifiedcopies.orders.api.util.FieldNameConverter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Implements validation of the request payload specific to the the create item request only.
+ */
+@Component
+public class CreateItemRequestValidator extends RequestValidator {
```

Comment:
`CreateItemRequestValidator`. Given this validates certified copy items, I would give it a more specific name.

Reply:
This has been renamed.